### PR TITLE
Top CRT: ADC -> PE conversion

### PR DESCRIPTION
**Pull request by francescopoppi**
This PR enables the conversion from ADC to PE using the correct values from calibration

### CODE REVIEW

**Review by tboone0042 — APPROVED** [review]
Looks fine to me, I see no issues, so you're just appending to the existing Side CRT database? Makes sense to me! I'm fine with merging names to just "CRT" or leaving space for the Top to make its own function, whatever the Top CRT Calibration Squad decides is best 😎

File: icaruscode/CRT/CRTUtils/CRTHitRecoAlg.cc [hunk 1]
```diff
@@ -520,8 +520,12 @@ sbn::crt::CRTHit CRTHitRecoAlg::MakeTopHit(
   double sum = 0;
   for (int chan = 0; chan < 32; chan++) {
     sum = sum + data->fAdc[chan];
-    float pe = (data->fAdc[chan] - ftopPed) / ftopGain;
+    std::pair<double, double> const chg_cal =
+              fChannelMap->getSideCRTCalibrationMap((int)data->fMac5, chan);
+    float pe = (data->fAdc[chan] - chg_cal.second) / chg_cal.first;
+    //float pe = (data->fAdc[chan] - ftopPed) / ftopGain;
     //      if(pe<=fPEThresh) continue;
+    if (pe<0) pe=0;
```
[the remark this answers is not in the document]
Comment (francescopoppi) [reply icaruscode/CRT/CRTUtils/CRTHitRecoAlg.cc:528]: the problem is petot += pe .
Doing: (ADC-PED)/Gain means that if the channel is not active in the trigger, its ADC value is going to be **_around_** the pedestal. This means that you could end up with  pes<0, therefore decrementing petot for no reason. This ensures that if a channel is around pedestal (slightly below, which is the most common case following an hardware problem we spotted), it is not destructive in the process of pe validation.

**Comment by francescopoppi** [comment]
Comments addressed

**Comment by mmrosenberg** [comment]
trigger build

**Review by PetrilloAtWork — APPROVED** [review]

**Pull request merged**